.github/workflows/e2e-test.yml

```diff
@@ -89,8 +92,40 @@ jobs:
           cd RhinoApp
           Rscript ../test-build-sass.R
 
+      - name: test_r() should run testthat tests
+        if: always()
+        run: |
+          cd RhinoApp
+          Rscript -e "rhino::test_r()"
+
       - name: test_e2e() should run cypress tests
         if: always()
         run: |
           cd RhinoApp
           Rscript -e "rhino::test_e2e()"
+
+      - name: Copy template files to RhinoApp
+        if: always()
+        shell: Rscript {0}
+        run: |
+          file.copy("app-files/main.R", "RhinoApp/app/main.R", overwrite = TRUE)
+          file.copy("app-files/say_hello.R", "RhinoApp/app/logic/say_hello.R", overwrite = TRUE)
+          file.copy("app-files/hello.R", "RhinoApp/app/view/hello.R", overwrite = TRUE)
+          file.copy("app-files/main.scss", "RhinoApp/app/styles/main.scss", overwrite = TRUE)
+          file.copy("app-files/index.js", "RhinoApp/app/js/index.js", overwrite = TRUE)
+          file.copy("app-files/test-hello.R", "RhinoApp/tests/testthat/test-hello.R", overwrite = TRUE)
+          file.copy("app-files/test-say_hello.R", "RhinoApp/tests/testthat/test-say_hello.R", overwrite = TRUE)
+          file.copy("app-files/hello.spec.js", "RhinoApp/tests/cypress/integration/hello.spec.js", overwrite = TRUE)
+          file.remove("RhinoApp/tests/testthat/test-main.R")
+
+      - name: Cypress tests should confirm RhinoApp works
+        if: always()
+        run: |
+          cd RhinoApp
+          Rscript -e "rhino::lint_r()"
+          Rscript -e "rhino::lint_sass()"
+          Rscript -e "rhino::lint_js()"
+          Rscript -e "rhino::build_sass()"
+          Rscript -e "rhino::build_js()"
+          Rscript -e "rhino::test_r()"
+          Rscript -e "rhino::test_e2e()"

```

Comment:
This actually confirms that `test_r()` works. `build_sass()` and `build_js()` are confirmed working by the cypress test.